### ISSUE TITLE
vboxnet0 is not found

### ISSUE THREAD

**Issue by Nikdoge** [GNS3/gns3-vm#102]
I'm getting this VirtualBox error, when trying to start GN3 VM directly from VirtualBox.

```
Could not start the machine ... because the following physical network interfaces were not found:
vboxnet0 (adapter 1)
You can either change the machine's network settings or stop the machine.
```
(It's english analogue; my error is in Russian) 

And indeed, there's no "vboxnet0" virtual host adapter. In my system it's called "VirtualBox Host-Only Ethernet Adapter". But why this virtual machine requires adapter, which does't exist?

OS: Windows 7
VirtualBox version: 5.1.30
GNS version: 2.1.0
GNS3 VM version: 2.1.0

===========================

I hope solution of this problem will solve my problem with GNS3 program, which is giving me next error:
```
ERROR __init__.py:289 Can't start the GNS3 VM: GNS3VM: DHCP must be enabled on VirtualBox host-only network: vboxnet0 for GNS3 VM
```
There's no way, you can enable DHCP on non-existent adapter, I think.

**Comment by blanosko** [comment]
Maybe too late but, I had same problem. Just rename network interface from "VirtualBox Host-Only Ethernet Adapter" to "vboxnet0". For me it worked

**Comment by grossmj** [comment]
The GNS3 VM is by default configured to use vboxnet0, however this interface doesn't exist by default on Windows ("VirtualBox Host-Only Ethernet Adapter" is the default). Now we automatically detect if vboxnet0 doesn't exist and try to replace it by the first available vboxnet on Windows. This fixes this issue.

**Comment by nameinsoft** [comment]
I have this problem too in ubuntu linux 18.04 please help me to solve that 

@grossmj 

Could not start the machine GNS3 VM because the following physical network interfaces were not found:

vboxnet0 (adapter 1)

You can either change the machine's network settings or stop the machine.

**Comment by bitlytanner** [comment]
Ignore the comment I made earlier. I was on the wrong repo entirely.

**Comment by PranavBhattarai** [comment]
I wanted to try GNS3VM in Virtual Box. Use the latest image on Fedora 36 and this is what I got:

![image](https://user-images.githubusercontent.com/32763112/175227694-bf8619a6-734f-45a1-b9ac-a36d79f223d7.png)

I thought the problem was resolved. Was it resolved for only Windows? I'm curious. @grossmj 

_I know VMware is usually recommended but ever since 5.16 kernel release, I'm seeing a lot of [this issue](https://communities.vmware.com/t5/VMware-Workstation-Pro/Compiling-vmmon-module-fails-on-linux-kernel-5-16-0-arch1-1/td-p/2887820) whenever there is a new small/major update in the kernel. Now we are on 5.18+, but still, there is no permanent fix yet from the VMware community. Although there is a workaround available. This is my I want to move on from VMware --->>> VirtualBox to get rid of this repetitive annoyance._
